components/content-container/readme.md

```diff
@@ -0,0 +1,15 @@
+This container element sets the maximum width of content on the page, centers the column horizontally in the viewport, and includes some padding that prevents its contents from butting against the edges of the viewport or against other adjacent elements. The maximum width and spacing change at different responsive breakpoints to adapt to different viewport widths.
+
+- The container's width isn't explicit; it's a fluid box with a minimum width of `304px` and constrained to a maximum width that differs at different responsive breakpoints. If you want to know how wide this box is, the answer is: it depends.
+- Side padding also adjusts at different breakpoints, with less padding in small windows and more padding in large windows.
+- You can apply theme classes to set a max-width narrower than the default, to constrain the maximum width even in larger viewports.
+    - `mzp-t-content-sm` : `432px`
+    - `mzp-t-content-md` : `688px`
+    - `mzp-t-content-lg` : `928px`
+    - `mzp-t-content-xl` : `1152px`
```

Comment:
Should I create variations for each of these classes? It would be neat to showcase all the different viewports!

Reply:
Update: added variants in my latest commit.